src/clib/pio.h

```diff
@@ -878,9 +1095,9 @@ extern "C" {
 
     /* Variables. */
     int PIOc_inq_varid(int ncid, const char *name, int *varidp);
-    int PIOc_inq_var(int ncid, int varid, char *name, nc_type *xtypep, int *ndimsp, int *dimidsp,
-                     int *nattsp);
-    int PIOc_inq_varname(int ncid, int varid, char *name);
+    int PIOc_inq_var(int ncid, int varid, char *name, int namelen, nc_type *xtypep, int *ndimsp,
+                     int *dimidsp, int *nattsp);
+    int PIOc_inq_varname(int ncid, int varid, char *name, int namelen);
```

Comment:
What's the idea behind introducing a namelen parameter? That's more suited to Fortran than C...